### ISSUE TITLE
Unable to pull Processing Sound library to Contribution Manager*

### ISSUE THREAD

**Issue by SEK-KAI** [processing/processing-sound#42]
*This might not be applicable, but I haven't found an answer to this anywhere for quite some time.

I've attempted to download the Sound library through the contribution manager through the Processing Development Environment and each time it is unable to initialize the download/installation process. I'm not entirely sure what would cause this, other than that the link to download doesn't work. Is there a common issue with downloading libraries via contribution manager?

I've also attempted to install manually, but haven't had much luck since the library cannot be found through the means I've tried. Does anyone else have issues downloading/installing this library?

**Comment by kevinstadler** [comment]
@SEK-KAI do you get a particular error message when trying to install the library? I just tried a clean download of the current version (2.2.2) using the Contribution Manager and it worked fine for me. Could it be that Processing does not have write access to the folder where the libraries should go? Do you have problems downloading any other Processing library?

If you want to try installing manually, grab the `sound.zip` file that is linked at the bottom of this page: https://github.com/processing/processing-sound/releases/tag/latest and extract it in the `libraries` folder that is inside the `Processing` folder (where your sketches are stored by default).

On my machine (a Mac) the folder structure of the installed library looks like this:
```
$ ls /Users/kevin/Documents/Processing/libraries/sound
LICENSE        README.md      examples       library        library.properties
```

**Comment by SEK-KAI** [comment]
@kevinstadler There's no real error message provided by the contribution manager. Upon clicking on install, it attempts to install for a couple of seconds and then the description updates to "Error during download and install of Sound." I've tried from a fresh version of Processing as well and still, I cannot install it. I found the Minim library and have installed it after I had trouble with the sound library.

I'm not sure what the difference between the .zip file you linked to and the one I tried manually installing before, but that worked. I've added libraries and other dependencies manually before,  but I've never had this much trouble. At least for now I have it functional inside the Processing IDE.
